Well, I am all out of ideas. At least I got the error message to change.

It now breaks trying to run `commitlint-travis`. 
```sh
Error: Cannot find module 'babel-runtime/helpers/slicedToArray'
```

I will no be offended if someone else raises a PR to fix this, or commits directly to this one.

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,8 +4,7 @@ language: node_js
 node_js:
   - "lts/*"
 cache:
-  directories:
-    - "node_modules"
+  yarn: true
 
 env:
   global:
@@ -13,7 +12,9 @@ env:
     - SLS_DEBUG=true
 
 before_install:
-  - npm install -g serverless
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.6.0
+  - export PATH=$HOME/.yarn/bin:$PATH
+  - yarn global add serverless
 
 script:
   - commitlint-travis

